src/lib/corePeripherals.ts

```diff
@@ -50,15 +55,17 @@ export type TSR_DeviceType = number
 
 export interface InitOptions {
 	category: DeviceCategory
-	type: DeviceType
-	subType: DeviceSubType
+	type?: DeviceType | string
```

Comment:
Hmm... I'm not entirely sure if having type and subType as optional is a good idea. I think that at least type should be mandatory. The UI has facilities for restarting playout gateway from the RundownView page and the device type and subType is the only reliable way of finding out which device should be restarted.

Reply:
Instead of relying of subtype, I think there should be a `supportsRestart` or similar property to control that, to avoid core needing to understand the different types

Reply:
Well, I agree that ideally, core wouldn't have to understand the types, but:
- most of the devices support restarting them
- users are only interested in restarting the playout gateway from the RundownView

Reply:
Either a config for the UI facilities or a config option in the manifest, yeah. It's not the point of whether it can be restarted (all gw's have buttons for that, assuming they run in some kind of monitor that auto restarts) but whether it is shown in the RundownView.

The ultimate goal (if we ever reach it) is to completely decouple the gateways from the core. Perhaps it's a bit too early to make it optional now, but the intent is to discourage making new couplings. Playout-gw will not lose the property immediately though.

Reply:
Very true, that does sound like a task for a bit later then

Reply:
OK, I suppose this shouldn't cause any typing errors. Let's merge it.